.travis.yml

```diff
@@ -48,8 +64,7 @@ install:
   - pip install .[SparkTrials]
   - pip install .[ATPE]
   - pip install pytest>=3.6
-  - pip install pytest-cov pep8 pytest-pep8
-  - pip install wrapt
+  - pip install pytest-cov pep8 pytest-pep8 wrapt
```

Comment:
I wonder whether we could use `pyproject.toml` , see `https://www.python.org/dev/peps/pep-0518/#specification`, and forget about all these hardcoded deps @maxpumperla